### PR TITLE
Adds the mystical wardrobe

### PR DESCRIPTION
**Pull request by ghost**
The mystical wardrobe replaces the messy amount of robes strewn all around the wizard shuttle by providing a compact, convenient space for storage. By using the wardrobe, one can withdraw any type of robes, wizard hat, or magical shoes available in the storage. One can also attack the wardrobe with any magical clothing article to put it away (read: delete).

The wardrobe is placed to the left of the magic mirror.

### CODE REVIEW

File: code/game/objects/structures/wizard_wardrobe.dm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+					if("Red")
+						robe_type = /obj/item/clothing/suit/wizrobe/red
+					if("Yellow")
+						robe_type = /obj/item/clothing/suit/wizrobe/yellow
+					if("Magus Blue")
+						robe_type = /obj/item/clothing/suit/wizrobe/magusblue
+					if("Magus Red")
+						robe_type = /obj/item/clothing/suit/wizrobe/magusred
+					if("Yellow")
+						robe_type = /obj/item/clothing/suit/wizrobe/yellow
```
Comment (Anonus) [inline code/game/objects/structures/wizard_wardrobe.dm]: Yellow's here twice.

File: code/game/objects/structures/wizard_wardrobe.dm [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	if(ishuman(user))
+		var/mob/living/carbon/human/M = user
+		var/choice = input(user, "What do you want to take from the wardrobe?", "Clothes") as null|anything in list("Robes", "Hat", "Sandals")
+
+		if(!user.canUseTopic(src))
+			return
+		switch(choice)
+			if("Robes")
+				var/robe_type = null
+				var/robe_color = input(user, "What variety?", "Clothes") as null|anything in list("Blue", "Red", "Magus Blue", "Magus Red", "Yellow", "Witch")
```
Comment (RemieRichards) [inline code/game/objects/structures/wizard_wardrobe.dm]: "What variety?" doesn't really make sense. "What type?" or "What outfit?" does.

File: code/game/objects/structures/wizard_wardrobe.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/objects/structures/wizard_wardrobe.dm]: What the _fuck_. I fixed that at least three times. Thanks for the heads-up, I'll fix it.

**Comment by Incoming5643** [comment]
Now we just need a jesus lion

**Comment by Cheridan** [comment]
Doesn't look like you used the map cleaner.

**Comment by Xhuis** [comment]
how do i

**Comment by Jordie0608** [comment]
https://tgstation13.org/wiki/Map_Merger

**Comment by Xhuis** [comment]
Well, map cleaner didn't do a whole lot. I did the setup, made my changes from scratch, and then ran the cleaner on the file - still no dice.

**Comment by Jordie0608** [comment]
Just keep trying; get someone to guide you along step by step in coderbus and you'll find whats not working right.

**Comment by Xhuis** [comment]
Okay, scrapped the original remodelling because the map merger wanted me to cry tears of blood. I just slapped it down next to the magic mirror. This is merge-ready.

**Comment by Xhuis** [comment]
w h y

**Comment by Xhuis** [comment]
Fuck this

**Pull request closed without merging**